### DISCUSSION TITLE
build for SUSE Linux Enterprise SLES

### DISCUSSION THREAD

**Discussion by Tomefa** [cachewerk/relay#106] [Support] [answered]
Hi,

we would like to try relay on our environment but i cannot find a way to get the PHP extension on SUSE Linux Enterprise (SLES).

Would it be possible to add this system on the build ?

Thanks

**Comment by tillkruss** [comment]
Sure thing! I'm not familiar with SUSE.

- What architecture are you using `x86`?
- What version of SLES are you running? 

Can you point us at the correct Docker image we can use for verifying the builds?

Anything on https://registry.suse.com, or https://hub.docker.com/r/opensuse/leap ?

**Reply by szepeviktor** [reply]
Tried the first image.
https://github.com/szepeviktor/enterprise-suse-image/actions/runs/5905900502/job/16020862034
It was successful.

**Reply by szepeviktor** [reply]
I think SUSE Linux Enterprise is in `uses: docker://registry.suse.com/suse/sle15:latest`

In the Dockerfile
```dockerfile
FROM registry.suse.com/suse/sle15:latest
```

**Comment by Tomefa** [comment]
Hi @tillkruss, we are using SLE 15, like @szepeviktor says:
`docker://registry.suse.com/suse/sle15:latest`

**Reply by tillkruss** [reply, marked as answer]
It looks like SLES is compatible with EL8, which we already have builds for. We'll add a Docker example shortly to illustrate the installation: https://github.com/cachewerk/relay/pull/108/files

In the meantime you could just use the "CentOS 8" builds: https://relay.so/builds

**Reply by tillkruss** [reply]
We've merged the working Docker example: https://github.com/cachewerk/relay/blob/main/docker/sle15.Dockerfile
